add_data.py

```diff
@@ -1,49 +1,60 @@
-import weaviate
 import pandas as pd
+import weaviate
+import weaviate.classes as wvc
+from weaviate import Config
+
+COLLECTION_NAME = 'Comments'
+# Set up the client
+client = weaviate.Client("http://localhost:8080",
+                         additional_config=Config(grpc_port_experimental=50051),
+                         )
+
+# If collection already exists, delete it
+if client.collection.exists(COLLECTION_NAME):
+    client.collection.delete(COLLECTION_NAME)
+
+# Create collection
+client.collection.create(
+    name=COLLECTION_NAME,
+    properties=[
+        wvc.Property(
+            name="comment",
+            data_type=wvc.DataType.TEXT,
+            descpirtion="The text of the comment"
```

Comment:
I notice that there is a typo here. Does this throw an error?

```suggestion
            description="The text of the comment"
```

Reply:
thanks! 
no, it doesn't throw an error